packages/renderer/package.json

```diff
@@ -6,8 +6,16 @@
   "types": "index.d.ts",
   "author": "Diego Muracciole <diegomuracciole@gmail.com>",
   "homepage": "https://github.com/diegomura/react-pdf#readme",
+  "type": "module",
   "main": "./lib/react-pdf.cjs.js",
   "module": "./lib/react-pdf.es.js",
+  "exports": {
+    ".": {
+      "import": "./lib/react-pdf.es.js",
```

Comment:
Note: we also need to add `types` there, as types are not siblings of the outputted JS files.

Reply:
How would this affect backwards compatibility? I know some module resolvers do not work with exports yet right?